decidim-core/app/views/layouts/decidim/header/_redesigned_main.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="relative w-full">
+  <div class="container flex justify-between items-center py-5">
+    <div class="w-3/4 md:w-2/5 lg:w-1/4 prose prose-img:max-h-16 prose-img:my-0">
+      <%= render partial: "layouts/decidim/logo", locals: { organization: current_organization } %>
+    </div>
+    <div class="hidden md:block md:w-2/5 lg:w-1/6">
+      <%= render partial: "layouts/decidim/header/redesigned_main_search" %>
+    </div>
+    <div class="hidden lg:flex justify-between text-center divide-x-2 divide-gray-3 ml-auto">
+      <%= render partial: "layouts/decidim/header/redesigned_main_links_desktop" %>
+    </div>
+
+    <div class="lg:hidden">
+      <%# Sticky menu mobile %>
+      <div class="fixed bottom-0 left-0 z-20 bg-white w-full px-4 py-3 grid grid-cols-4 text-secondary box-[0_-4px_6px_rgba(198,198,198,0.25)]">
+        <%= render partial: "layouts/decidim/header/redesigned_main_links_mobile" %>
```

Comment:
We should always avoid duplicating the same content just when it has been displayed and rethink the DOM structure so that duplication would not be necessary. Some assistive users may disable the CSS of the page (e.g. blind users) when they would "see" (or hear) two navigations in a row.

Foundation handles this currently so that it moves the content dynamically to different DOM sections when the breakpoints are hit. It's not optimal either but better than duplication.

Another comment is that there is a hard-coded color here in the HTML (above this line).